publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/NonDegreePermissionStrategyTest.java

```diff
@@ -0,0 +1,151 @@
+                              .build();
+
+        var requestInfo = createUserRequestInfo(curatorUsername, institution, getCuratorAccessRightsWithDegree(),
+                                                cristinId, publication.getResourceOwner().getOwnerAffiliation());
+        var userInstance = RequestUtil.createUserInstanceFromRequest(requestInfo, identityServiceClient);
+
+        Assertions.assertFalse(PublicationPermissionStrategy
+                                  .create(publication, userInstance, uriRetriever)
+                                  .allowsAction(PublicationOperation.UPDATE));
+    }
+
+    @Test
+    void shouldAllowCuratorWithEmbargoAccess()
+        throws JsonProcessingException, UnauthorizedException {
+
+        var institution = randomUri();
+        var resourceOwner = randomString();
+        var curatorUsername = randomString();
+        var cristinId = randomUri();
+
+        var publication = createDegreePhd(resourceOwner, institution, randomUri()).copy()
+                              .withStatus(PUBLISHED)
+                              .withAssociatedArtifacts(List.of(randomFileWithEmbargo(), PublishedFileGenerator.random()))
+                              .build();
+
+        var requestInfo = createUserRequestInfo(curatorUsername, institution, getCuratorAccessRightsWithEmbargoDegree(),
+                                                cristinId, publication.getResourceOwner().getOwnerAffiliation());
+        var userInstance = RequestUtil.createUserInstanceFromRequest(requestInfo, identityServiceClient);
+
+        Assertions.assertTrue(PublicationPermissionStrategy
+                                   .create(publication, userInstance, uriRetriever)
+                                   .allowsAction(PublicationOperation.UPDATE));
+    }
+
+    @Test
+    void shouldDenyDegreeCuratorEmbargoAccess()
+        throws JsonProcessingException, UnauthorizedException {
+
+        var institution = randomUri();
+        var resourceOwner = randomString();
+        var curatorUsername = randomString();
+        var cristinId = randomUri();
+
+        var publication = createDegreePhd(resourceOwner, institution, randomUri()).copy()
+                              .withStatus(PUBLISHED)
+                              .withAssociatedArtifacts(List.of(randomFileWithEmbargo(), PublishedFileGenerator.random()))
+                              .build();
+
+        var requestInfo = createUserRequestInfo(curatorUsername, institution, getCuratorAccessRightsWithDegree(),
+                                                cristinId, publication.getResourceOwner().getOwnerAffiliation());
+        var userInstance = RequestUtil.createUserInstanceFromRequest(requestInfo, identityServiceClient);
+
+        Assertions.assertFalse(PublicationPermissionStrategy
+                                  .create(publication, userInstance, uriRetriever)
+                                  .allowsAction(PublicationOperation.UPDATE));
+    }
```

Comment:
Duplicate of the test above as far as I can see. Only difference being the addition of a non embargo file in associated artifacts. Adding this file to the test on line 74 would give as much cover I would presume?

```suggestion
```